autotracking/src/main/java/com/tealium/autotracking/internal/PushNotificationAutoTracker.kt

```diff
@@ -0,0 +1,42 @@
+package com.tealium.autotracking.internal
+
+import com.google.firebase.messaging.RemoteMessage
+import com.tealium.autotracking.PushNotificationTracker
+import com.tealium.core.TealiumContext
+import com.tealium.dispatcher.TealiumEvent
+
+class PushNotificationAutoTracker(private val context: TealiumContext) : PushNotificationTracker {
+
+    private val PUSH_NOTIFICATION_PREFIX: String = "push_notification"
```

Comment:
I believe `private val` constants should be either top-level file constants or in a companion object.